### PR TITLE
fix: Show exact start date on events page

### PR DESCRIPTION
**Pull request by ombharatiya**
## Context

Exact “Start Date” is not available on events page, only English approximation, so can’t easily compare against other events

## Objective

Show Exact Date Time in MM/DD/YYYY hh:mm AM/PM format for Start Date on Events Page

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by jithine** [comment]
Can you add screenshots showing the change ? Tests are still failing, think it's one of the flaky ones, we need to fix them.

**Comment by ombharatiya** [comment]
Adding some screenshots for the reference about the change:

Image 1: Date Time Format On Pipeline Home Page & Cards in Events Tab

Screenshot 2021-07-09 at 8 09 42 PM

Image 2: Date Time Format On Pipeline Home Page & Cards in  PR Tab

Screenshot 2021-07-09 at 8 10 04 PM

**Comment by tkyi** [comment]
Should we add `UTC` at the end of the time text?

**Comment by jithine** [comment]
It should be browser's local time not utc. I think it's a good idea to display TZ, looks like https://github.com/screwdriver-cd/ui/blob/master/app/utils/time-range.js#L23 already supports an option for it.

File: tests/integration/components/pipeline-list-coverage-cell/component-test.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ module('Integration | Component | pipeline-list-coverage-cell', function(hooks)
     assert.equal(find('.coverage').textContent.trim(), 'N/A');
   });
 
-  test('it renders with actual coverage value', async function(assert) {
+  test('it renders with no coverage value', async function(assert) {
```
Comment (tkyi) [inline tests/integration/components/pipeline-list-coverage-cell/component-test.js:43]: should not change this test

File: app/utils/time-range.js [hunk 2]
```diff
@@ -95,3 +95,19 @@ export const CONSTANT = {
...
+//  * @return {String} human readable text for date time
+//  */
+//  export function dateTimeText(dateTime, timeZone) {
+
+//   if (!dateTime) {
+//     return '--';
+//   }
+
+//   return `${toCustomLocaleString(new Date(dateTime), { timeZone })}`;
+// }
```
Comment (tkyi) [inline app/utils/time-range.js:113]: commented code?

File: tests/integration/components/pipeline-list-coverage-cell/component-test.js (hunk 1 above)
Comment (ombharatiya) [reply tests/integration/components/pipeline-list-coverage-cell/component-test.js:43]: Sure, was tweaking to check any workaround on this test scenario.

**Comment by tkyi** [comment]
Two comments:

1. I think ideally, users could have the option to have the human-readable words or more exact time like we have for build logs
If you click the upper left hand corner in the logs window, you can cycle between timestamp options to show: `UTC`, `local time`, etc
Screen Shot 2021-07-16 at 11 28 09 AM
Screen Shot 2021-07-16 at 11 28 03 AM

2. There are few countries that use MM/DD/YYYY system (US, the Philippines, Palau, Canada, and Micronesia). Should we be using a more universal format?
Seems like it's hardcoded to US locale: https://github.com/screwdriver-cd/ui/blob/master/app/utils/time-range.js#L42

**Comment by jithine** [comment]
For 2. We will have to overhaul our UI. Right now let it be hard coded to "en-US". We will eventually have to support locale region user preference in UI.

**Review by jithine — APPROVED** [review]

**Review by adong — APPROVED** [review]
Looks good to me.

**Comment by adong** [comment]
This PR shows the exact time in local time format, and I think we can let go of the switching of time formats _for now_.

**Pull request merged**